Makefile

```diff
@@ -81,7 +81,7 @@ clean:
 
 .PHONY: clean-powerwash
 clean-powerwash: clean
+	npx rimraf *storageState.json test-results
 	docker system prune -f --volumes
 	docker rmi -f `docker images -q "lf-*"`
-	docker image rm sillsdev/web-languageforge:base-php
```

Comment:
are you sure this is actually cleaned up with the "lf-*" ?  I think the image name is still `base-php` and you haven't renamed it in this PR AFAICT.

Reply:
I can now see that base-php image isn't normally on one's system, unless the dev expressly builds the image (which is still a possibility and was true on my system).  I'd like to leave this line in place since some devs might build this image and expect a powerwash to clean it up

Reply:
fwiw, if you leave this removal in place for everyone, everyone will see a failure every time they run the target, 

![image](https://user-images.githubusercontent.com/4412848/198859884-ef6369eb-7147-40aa-bfa9-cb8f365599f3.png)

whereas simply recognizing a local build of `base-php` as a special case and relying on the individual to run a special command, i.e., `docker rmi <image id>` seems appropriate to me.

Reply:
ah, ok, I see now that base-php only exists on dev machines that explicitly build it.  How about I make the line only execute if the image exists?